parsl/executors/high_throughput/executor.py

```diff
@@ -435,8 +435,7 @@ def outstanding(self):
 
     @property
     def connected_workers(self):
-        workers = self.command_client.run("MANAGERS")
-        # logger.debug("Got managers: {}".format(workers))
+        workers = self.command_client.run("WORKERS")
```

Comment:
A quick look is that, I think, we should have a new method `connected_workers`, and change the original `connected_workers` to `connected_managers`, instead of replacing.

This is because the method `_hold_block` below apparently uses `connected_workers` as the context of # of connected managers. There may be other places that use the same context.

Reply:
Great catch, @ZhuozhaoLi. This bit of code was merged recently. I was working on an outdated branch and didn't catch this when I rebased to master. I'll create a new method as you suggest.